media-libs/wxsvg/wxsvg-1.5.22.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+WX_GTK_VER=3.0
+inherit eutils wxwidgets
+
+DESCRIPTION="C++ library to create, manipulate and render SVG files"
+HOMEPAGE="http://wxsvg.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.bz2"
+
+LICENSE="wxWinLL-3"
+SLOT="0/3" # based on SONAME of libwxsvg.so
+KEYWORDS="~amd64 ~x86"
+IUSE="static-libs libav"
+
+RDEPEND=">=dev-libs/expat-2:=
+	media-libs/libexif:=
+	>=dev-libs/glib-2.28:2=
+	dev-libs/libxml2:=
+	media-libs/fontconfig:=
+	media-libs/freetype:2=
+	x11-libs/cairo:=
+	x11-libs/pango:=
+	x11-libs/wxGTK:${WX_GTK_VER}=[X]
+	libav? ( >=media-video/libav-9:0= )
```

Comment:
libav is gone.

Reply:
> libav is gone. It may have been better to depend on virtual/ffmpeg in the past though.

So should I remove libav and only have the option for ffmpeg?
Or is better to have virtual/libav for transition purposes?

Reply:
virtual/ffmpeg is gone, so just depend on ffmpeg with whatever USE you need, if any. Nice and simple!